messages/Exceptions.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I thought that we could start to consolidate exception messages into a single file. A message file per .ts file seemed to be getting unwieldy.